request/common.go

```diff
@@ -0,0 +1,30 @@
+package request
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func normalizePasswordStorePath(storePath string) (string, error) {
+	if strings.HasPrefix(storePath, "~/") {
+		storePath = filepath.Join("$HOME", storePath[2:])
+	}
+	storePath = os.ExpandEnv(storePath)
+
+	directStorePath, err := filepath.EvalSymlinks(storePath)
```

Comment:
What is the reason for dereferencing symlinks here?

Reply:
The purpose of this method is to dereference _everything_ (`~/`, environment variables, symlinks), and then check that the final destination exists and is a directory. It's like a safeguard to make sure that whatever user configured is the right thing, and that in the future this app will be able to list files at this location.